cassandra-analytics-integration-tests/src/test/java/org/apache/cassandra/analytics/SparkTestUtils.java

```diff
@@ -167,21 +173,27 @@ public SparkConf defaultSparkConf()
 
     public void validateWrites(List<Row> sourceData, Object[][] queriedData)
     {
+        validateWrites(sourceData, queriedData, null, Row::getInt);
```

Comment:
can we pass the `DEFAULT_COLUMNS_MAPPER`, instead of `null`? and remove the null check in the method below?

Reply:
we pass `null` from the test class as well. I wanted to avoid passing it from the test class. But I guess we can disallow nulls here instead